### PR TITLE
Header

### PR DESCRIPTION
**Pull request by SolivanLau**
Hello! Mostly worked through global html and css for header nav and favicon in head. 
I have applied changes to current html pages globally. `page-template.html` has a clean copy of header with just navigation.

### Global FavIcon
- added file and added for most, if not all current pages

### Header Nav Styles
- worked through weird text wrap issue ~ 1400px
- inline svgs have been updated to img src svgs
- aria label changes for mobile menu btn via js
- support btn now shows up in  mobile menu
- corrected web copy "US". will change Support btn to Contact on monday

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -41,58 +45,37 @@ <h1 class="sr-only">If These Lands Could Talk</h1>
...
-									/>
-								</svg>
+								/>
 								<!-- exit menu icon -->
-								<svg
-									width="24"
-									height="24"
-									viewBox="0 0 24 24"
-									xmlns="http://www.w3.org/2000/svg"
+								<img
```
Comment (SolivanLau) [inline about.html:57]: Now using img src instead of inline svg

File: about.html [hunk 2]
```diff
@@ -3,7 +3,12 @@
 	<head>
 		<meta charset="UTF-8" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<title>If These Lands Could Talk - Page Name</title>
+		<link
```
Comment (SolivanLau) [inline about.html:6]: favicon

File: js/main.js [hunk 3]
```diff
@@ -160,6 +160,11 @@ mobileMenuBtn.addEventListener("click", () => {
 	menuIconOpen.classList.toggle("hidden");
 	menuIconClose.classList.toggle("hidden");
 
+	if (!menuIconOpen.classList.contains("hidden")) {
```
Comment (SolivanLau) [inline js/main.js:163]: updating aria label after mobile menu icons and buttons are toggled

File: css/global.css [hunk 4]
```diff
@@ -733,18 +733,14 @@ header nav .flex-container {
 	display: flex;
 	justify-content: space-between;
 	align-items: center;
-	gap: 65px;
+	gap: var(--spacing-sm);
```
Comment (SolivanLau) [inline css/global.css:736]: using smaller gap for when flex items jumble up together.

File: css/global.css [hunk 5]
```diff
@@ -900,13 +901,15 @@ Media Queries (overriding styles for different screen sizes)
 ===================================
 */
 
-/* custom breakpoint for top navigation 1360px*/
-@media screen and (max-width: 1360px) {
+@media screen and (max-width: 1395px) {
```
Comment (SolivanLau) [inline css/global.css:904]: custom mq to deal w nav link sizing and font-size

File: about.html (hunk 2 above)
Comment (maxharrisnet) [reply about.html:6]: 😌

File: about.html [hunk 6]
```diff
@@ -106,7 +89,7 @@ <h1 class="sr-only">If These Lands Could Talk</h1>
 				<!-- Wrapper -->
 				<div class="wrapper hero-wrapper">
 					<div class="hero-text-container">
-						<h2>If These Lands Could Talk... What Would They Say?</h2>
+						<h1>If These Lands Could Talk... What Would They Say?</h1>
```
Comment (maxharrisnet) [inline about.html:92]: 👂🏾 🍃

File: about.html (hunk 1 above)
Comment (maxharrisnet) [reply about.html:57]: Inline SVG alway feels like  a good idea at the time, and then it's sand in your pants 🐶

File: css/global.css (hunk 5 above)
Comment (maxharrisnet) [reply css/global.css:904]: In my experience, the navigation is usually where you have to break out some funky custom breakpoints. This works!

File: js/main.js (hunk 3 above)
Comment (maxharrisnet) [reply js/main.js:163]: Nice!

**Review by maxharrisnet — APPROVED** [review]

**Comment by maxharrisnet** [comment]
@SolivanLau I approved most of your PRs. I am not sure what the best order is for merging them in. Please merge the approved PRs in as you see fit.

File: about.html (hunk 1 above)
Comment (SolivanLau) [reply about.html:57]: if only we were working with functional components :')

**Pull request merged**